transcendence-app/src/chat/chat.controller.ts

```diff
@@ -380,4 +389,40 @@ export class ChatController {
     }
     return messages;
   }
+
+  @Put('room/:chatroomId/avatar')
+  @ApiConsumes('multipart/form-data')
+  @ApiFile('file')
+  @ApiProduces('image/jpeg')
+  @ApiOkResponse({
+    description: 'Update a chatroom avatar',
+    type: AvatarResponseDto,
+  })
+  @ApiNotFoundResponse({ description: 'Not Found' })
+  @ApiUnprocessableEntityResponse({ description: 'Unprocessable Entity' })
+  @ApiPayloadTooLargeResponse({ description: 'Payload Too Large' })
```

Comment:
Is this decorator used automatically with a certain file size? Where is defined? Nestjs, Db, server?